README.md

```diff
@@ -461,6 +461,23 @@ bundle exec rake console
 => Reek::Examiner
 ```
 
+You can also use Pry while running the tests by adding the following at the
+point where you want to start debugging:
+
+```ruby
+require 'pry'
+binding.pry
+```
+
+If you do this, please run the specs with `DEBUG=1` set, e.g.:
```

Comment:
Let's not use words like please in this context, otherwise we would have to use them all over the place ;)